dashboard/src/app/(authenticated)/(diagram)/components/NodeTypes/Sleep/SleepTaskDetails.tsx

```diff
@@ -0,0 +1,22 @@
+import { NodeRun, SleepNode } from 'littlehorse-client/proto'
+import { FC } from 'react'
+import { NodeDetails } from '../NodeDetails'
+import { convertSecondsToReadableTime } from '../../../../../utils'
+
+export const SleepTaskDetails: FC<{ sleepNode?: SleepNode; nodeRun?: NodeRun }> = ({ sleepNode, nodeRun }) => {
+  if (!sleepNode?.rawSeconds?.literalValue?.int) return
+
+  return (
+    <NodeDetails>
+      <div className="mb-2">
+        <div className="flex items-center gap-1 whitespace-nowrap text-nowrap">
+          <h3 className="font-bold">SleepTask</h3>
```

Comment:
Same here it should be just Sleep

Reply:
i think it ok to be called <NodeDetail> bcz its a wrapper for all nodes when user click on node

Reply:
I agree with @mijailr, it's not a task it should be renamed

File Name: `SleepTaskDetails.tsx` `SleepDetails.tsx`
<h3>: `SleepTask` -> `Sleep`